com.unity.shadergraph/Editor/Data/Util/DictionaryPool.cs

```diff
@@ -1,22 +1,18 @@
 using System.Collections.Generic;
+using UnityEngine.Pool;
 
 namespace UnityEditor.Graphing
 {
     static class DictionaryPool<TKey, TValue>
```

Comment:
There's already a `DictionaryPool<TKey, TValue>` in `UnityEngine.Pool` that we could use instead, and then get rid of this whole file.

Reply:
Done in latest commit.